R/learner_prioritylasso_classif_priority_lasso.R

```diff
@@ -13,7 +13,25 @@
 #' `r format_bib("klau2018priolasso")`
 #'
 #' @template seealso_learner
-#' @template example
+#' @examples
+#' if (requireNamespace("prioritylasso", quietly = TRUE)) {
+#' # Define the Learner and set parameter values
+#'
+#'
+#' learner = lrn("classif.priority_lasso", type.measure = "auc",
+#'   blocks = list(bp1 = 1:4, bp2 = 5:9, bp3 = 10:28, bp4 = 29:1028))
+#' print(learner)
+#'
+#' # Define a Task
+#' task = as_task_classif(prioritylasso::pl_data, target = "pl_out")
+#'
+#' # Train the learner on the training ids
+#' learner$train(task)
+#'
+#' # print the model
+#' print(learner$model)
```

Comment:
why no pred?

Reply:
no suitable task found

Reply:
why is training possible but no prediction? this seems to be weird